.travis.yml

```diff
@@ -44,14 +44,23 @@ script:
 
   elif [ "$TEST_TYPE" = build_website ]
   then
-
+    DOCS_VERSION=${TRAVIS_BRANCH%-stable}
```

Comment:
this is getting really bad to do it in a travis.yml file, I'd love if you just ported it to js :)